futaba/cogs/moderation/core.py

```diff
@@ -56,7 +56,7 @@ def build_reason(ctx, action, minutes, reason, past=False):
             full_reason.write(f" with reason: {reason}")
         return str(full_reason)
 
-    async def remove_roles(self, ctx, member, minutes, action, reason):
+    async def remove_roles(self, message, member, minutes, action, reason):
```

Comment:
Same as above.
I would do what you did for `perform_jail` with this as well as other commands use this as well and use `ctx`.
That or change them to use message.

Reply:
ok